.travis.yml

```diff
@@ -89,7 +89,7 @@ jobs:
       if: type != pull_request AND (branch in (master, develop) OR tag is present)
       python: 3.6
       install:
-        - echo "Install"
+        - git submodule update --init --recursive
```

Comment:
did you actually see a problem requiring this? travis generally does this by default...

Reply:
overcorrected when trying to resolve a related issue